core/src/main/java/org/elasticsearch/bootstrap/Bootstrap.java

```diff
@@ -93,12 +93,14 @@ public static void initializeNatives(Path tmpFile, boolean mlockAll, boolean sec
                 throw new RuntimeException("don't run elasticsearch as root.");
             }
         }
-        
+
         // enable secure computing mode
         if (seccomp) {
             Natives.trySeccomp(tmpFile);
+        } else {
+            logger.warn("Not enabling seccomp because it was explicitly disabled");
         }
```

Comment:
Can we remove the whitespace changes and the warning? I would also not like to document this in TESTING.md or anywhere else. It defeats the whole point of it. I also don't think we should support allowing tests to do this for any reason, but I know I stand alone.